recipes/psfex/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "psfex" %}
+{% set version = "3.21.1" %}
+
+package:
+  name: {{ name }}
```

Comment:
maybe name this `{{ name }}-c`

Reply:
The astromatic- prefix makes more sense if we are not going to change things. This way we can include sextractor, swarp etc in the same way for consistency.

Reply:
The other thing is that the dark energy survey has custom versions of these packages that I myself have been meaning to add using the des- prefix. At least this way everything would be consistent.

Reply:
Going slightly off topic here, but if DES requires custom versions they should perhaps not go into `conda-forge` but a separate `des` channel? 

I'm pushing this recipe as part of an effort to migrate not only our own Astro-WISE software but also the Euclid SGS processing environment over to conda environments. The idea is to use as vanilla 3rd party packages as much as possible. If we need modifications, I will try to get them in upstream. Failing that, we will put modified packages in our own channel for Astro-WISE (I'm not sure what the Euclid policy will be as I'm only remotely involved). 

Anyway, I will change this to `astromatic-{{ name }}`.